src/sybil/table_query.go

```diff
@@ -216,10 +216,10 @@ func (t *Table) LoadAndQueryRecords(flags *FlagDefs, loadSpec *LoadSpec, querySp
 				}
 			}(t)
 
-			if t.Options.SAMPLES {
+			if OPTS.SAMPLES {
 				wg.Wait()
 
-				if count > querySpec.Limit {
+				if querySpec != nil && count > querySpec.Limit {
```

Comment:
why was this added? (i'm curious about the context that led to needing this check)

Reply:
I think it was added before the outer nil check came into play, not necessary now.